### PR TITLE
OS v3.2.1 (E+ v9.5)

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Updates to OpenStudio v3.2.1 (EnergyPlus v9.5). See below for results.

## Checklist

Not all may apply:

- [x] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [x] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [x] No unexpected regression test changes
- [x] All tests are passing
- [x] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Comment by shorowit** [comment]
There is a bug in OpenStudio-standards such that propane boilers end up modeled as natural gas boilers: https://github.com/NREL/openstudio-standards/issues/1100

**Comment by shorowit** [comment]
CI results for ~250 project_national homes:

Result | Value
-- | --
Max Total Site Energy Change (MBtu) | 3.50
Min Total Site Energy Change (MBtu) | -3.10
Avg Total Site Energy Change (MBtu) | -0.06

If I force there to be no windows in the runs, the results are instead:

Result | Value
-- | --
Max Total Site Energy Change (MBtu) | 0.10
Min Total Site Energy Change (MBtu) | 0.00
Avg Total Site Energy Change (MBtu) | 0.01

So it appears that the EnergyPlus simple window model bugfix explains nearly all of the difference.

Breakout of changes by fuel use:

Fuel Use | Change (%)
-- | --
Electricity, total | 0.7%
Natural gas, total | -0.6%

Breakout of changes by end use:

End Use | Change (%)
-- | --
Electricity, cooling | 2.6%
Electricity, cooling fans | 2.1%
Electricity, heating fans | 1.8%
Electricity, ceiling fan | 1.0%
Electricity, heating | -2.3%
Natural gas, heating | -1.7%

**Comment by shorowit** [comment]
I believe that this is ready to go. @ejhw, @afontani what additional testing, if any, would you like?

**Review by ejhw — APPROVED** [review]
Thanks @shorowit ! The changes look sufficiently explained to me. It's up to @afontani whether we do a EULP run or wait and bundle with other changes.

**Comment by joseph-robertson** [comment]
We decided not to do a EULP run.

File: test/test_samples_osw/base_results/results.csv [hunk 1]
```diff
@@ -0,0 +1,19 @@
...l,misc_pool_heater,misc_pool_pump,misc_pool_schedule,misc_well_pump,natural_gas_clothes_dryer_therm,natural_gas_cooking_range_therm,natural_gas_fireplace_therm,natural_gas_grill_therm,natural_gas_heating_therm,natural_gas_hot_tub_heater_therm,natural_gas_lighting_therm,natural_gas_pool_heater_therm,natural_gas_water_systems_therm,natural_ventilation,neighbors,occupants,option_01_cost_usd,option_01_lifetime_yrs,orientation,overhangs,plug_load_diversity,plug_loads,propane_clothes_dryer_mbtu,propane_cooking_range_mbtu,propane_heating_mbtu,propane_water_systems_mbtu,pv,radiant_barrier,range_spot_vent_hour,refrigerator,refrigerator_schedule,roof_area_ft_2,roof_material_finished_roof,roof_material_unfinished_attic,setpoint_demand_response,size_cooling_system_kbtu_h,size_heating_system_kbtu_h,size_water_heater_gal,solar_hot_water,state,total_site_electricity_kwh,total_site_energy_mbtu,total_site_fuel_oil_mbtu,total_site_natural_gas_therm,total_site_propane_mbtu,total_site_wood_mbtu,vacancy_status,wall_area_above_grade_conditioned_ft_2,wall_area_above_grade_exterior_ft_2,wall_area_below_grade_ft_2,water_heater_efficiency,window_area_ft_2,window_areas,windows,wood_heating_mbtu,workflow_json
```
Comment (joseph-robertson) [inline test/test_samples_osw/base_results/results.csv:1]: @shorowit I don't think this file is supposed to be in here.

**Review by joseph-robertson — APPROVED** [review]

**Pull request merged**